<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetGamePlayer</class>
 <widget class="QWidget" name="WidgetGamePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>157</width>
    <height>181</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>157</width>
    <height>181</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>157</width>
    <height>181</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelKingdom">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="LabelClickable" name="labelRole">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>0</y>
     <width>25</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelNick">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>1</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelKingdomFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>15</y>
     <width>36</width>
     <height>98</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="LabelClickable" name="labelRoleZhong">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>27</y>
     <width>25</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="LabelClickable" name="labelRoleFan">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>53</y>
     <width>25</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="LabelClickable" name="labelRoleNei">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>80</y>
     <width>25</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelWuQi">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>124</y>
     <width>11</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="WidgetZhuangBeiFull" name="widgetWuQi" native="true">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>122</y>
     <width>150</width>
     <height>14</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetZhuangBeiFull" name="widgetFangJu" native="true">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>136</y>
     <width>150</width>
     <height>14</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetZhuangBeiFull" name="widgetJiaYiMa" native="true">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>150</y>
     <width>150</width>
     <height>14</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetZhuangBeiFull" name="widgetJianYiMa" native="true">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>164</y>
     <width>150</width>
     <height>14</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetZhuangBeiSmall" name="widgetJiaYiMaSmall" native="true">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>150</y>
     <width>75</width>
     <height>14</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetZhuangBeiSmall" name="widgetJianYiMaSmall" native="true">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>150</y>
     <width>75</width>
     <height>14</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHands">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>94</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHandsNumber">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>94</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBloodBg">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>labelBloodBg</zorder>
  <zorder>labelKingdom</zorder>
  <zorder>labelRole</zorder>
  <zorder>labelNick</zorder>
  <zorder>labelKingdomFrame</zorder>
  <zorder>labelRoleZhong</zorder>
  <zorder>labelRoleFan</zorder>
  <zorder>labelRoleNei</zorder>
  <zorder>labelWuQi</zorder>
  <zorder>widgetWuQi</zorder>
  <zorder>widgetFangJu</zorder>
  <zorder>widgetJiaYiMa</zorder>
  <zorder>widgetJianYiMa</zorder>
  <zorder>widgetJiaYiMaSmall</zorder>
  <zorder>widgetJianYiMaSmall</zorder>
  <zorder>labelHands</zorder>
  <zorder>labelHandsNumber</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelClickable</class>
   <extends>QLabel</extends>
   <header>ui/others/labelclickable.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>WidgetZhuangBeiFull</class>
   <extends>QWidget</extends>
   <header>ui/widget/widgetzhuangbeifull.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetZhuangBeiSmall</class>
   <extends>QWidget</extends>
   <header>ui/widget/widgetzhuangbeismall.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
